src/Containers/AddTodo/Action.js

```diff
@@ -0,0 +1,13 @@
+let nextTodoId = 0;
```

Comment:
Is it a good ideia? I think it should be place in a state instead of a global variable.